content/cpp/concepts/errors/errors.md

```diff
@@ -0,0 +1,88 @@
+---
+Title: 'Errors'
+Description: 'Errors make it easier to locate and correct problems within the source code.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Classes'
+  - 'Objects'
+  - 'Constructors'
+  - 'OOP'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+Running into an error is common when using C++. Error messages and the types associated with them make it easier to locate and correct a problem.
+
+Some common types of errors are:
+
+- Syntax Errors
+- Run-Time Errors
+- Link-Time Errors
+- Logic Errors
```

Comment:
Since we're going into detail about each of these error types, what do we think about replacing these lines with 1-2 sentences pertaining to how there are different kinds for different purposes and placing next to the text on line #17?